src/+otp/+qg/QuasiGeostrophicProblem.m

```diff
@@ -166,7 +176,7 @@ function onSettingsChanged(obj)
             F = sin(pi*(ymat.' - 1));
             
             obj.Rhs = otp.Rhs(@(t, psi) ...
-                otp.qg.f(psi, Lx, Ly, P1, P1T, P2, P2T, L12, Dx, Dy, F, Re, Ro), ...
+                otp.qg.f(psi, Lx, Ly, P1, P1T, P2, P2T, L12, Dx, DyT, F, Re, Ro), ...
```

Comment:
Is it really necessary to pass in P1T and P2T?  It is more storage and arguments and I wouldn't think it's more efficient than transposing in f.

Reply:
I've timed it, and it is in fact faster to pre-compute the transposes. I am going to replace this with fast sine transforms once I figure out the correct way to do that anyways, so this will not exist in the final pull request (this is a draft).

Reply:
Actually in timings, this is faster than doing an fft. I assume that has a lot of overhead, so I will keep this version for now.